app/src/main/java/org/sopt/and/ui/screen/signIn/viewmodel/SignInViewModel.kt

```diff
@@ -0,0 +1,56 @@
+package org.sopt.and.ui.screen.signIn.viewmodel
+
+import android.content.Context
+import android.widget.Toast
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import androidx.lifecycle.ViewModel
+import androidx.navigation.NavHostController
+import dagger.hilt.android.lifecycle.HiltViewModel
+import org.sopt.and.data.UserManager
+import javax.inject.Inject
+
+@HiltViewModel
+class SignInViewModel @Inject constructor(
+    private val userManager: UserManager
+) : ViewModel() {
```

Comment:
아래와 같이 바꿔서 context를 전달받지 않게 해도 좋을 것 같아요~!!
```suggestion
class SignInViewModel @Inject constructor(
    private val userManager: UserManager,
    @ApplicationContext private val myContext: Context
) : ViewModel() {
```

Reply:
덕분에 코드가 훨씬 간결해졌습니다. 꿀팁 감사합니다.